products/llbuildSwift/CoreBindings.swift

```diff
@@ -237,6 +237,18 @@ public protocol BuildEngineDelegate {
     /// requested Key cannot be supplied, the delegate should provide a dummy rule
     /// that the client can translate into an error.
     func lookupRule(_ key: Key) -> Rule
+
+    /// Handle error message reported by the engine
+    ///
+    /// The delegate should handle this, but a default implementation is
+    /// is provided that does nothing for backward compatiblity.
+    func error(_ message: String)
+}
+
+extension BuildEngineDelegate {
+    func error(_ message: String) {
```

Comment:
Shouldn't this be public?

Reply:
Probably.  Smoke tests seem to indicate so.  Wonder why my local tests passed? 🤔